packages/@okta/vuepress-site/docs/reference/rl-additional-limits/index.md

```diff
@@ -81,13 +80,25 @@ Limits are applied on a per-recipient basis and vary by email type. The limit fo
 
 ### Per-user limits
 
-API endpoints that take username and password credentials, including the [Authentication API](/docs/reference/api/authn/) and the [OAuth 2.0 resource owner password flow](/docs/guides/implement-password/), have a per-username rate limit to prevent brute force attacks with the user's password:
+API endpoints that take username and password credentials, including the [Authentication API](/docs/reference/api/authn/), the [OAuth 2.0 resource owner password flow](/docs/guides/implement-password/), have a per-username rate limit to prevent brute force attacks with the user's password:
 
 | Action and Okta API endpoint                                      | Per User limits (all orgs) |
 | ----------------------------------------------------------------- | -------------------------: |
 | **Authenticate the same user:**<br>`/api/v1/authn`                | 4 per second               |
 | **Generate or refresh an OAuth 2.0 token:**<br>`/oauth2/v1/token` | 4 per second               |
 
+SMS and Call factor endpoints also have a per-username rate limit:
+
+#### 30 second verification rate limits
```

Comment:
This applies to both verificaton and enrollment. I am wondering if something along the lines of `per user/per device rate limit` will be a more accurate title. 
@ericcarroll-okta thoughts?

Reply:
It's more `per phone number` or `per phone` than `per device`.  It isn't tied to a `device` in the same way OV is tied to its device.  For example:  Your phone number can be moved.  I'd let OV keep `device` and not reuse its term.